geoengine_bing/geo_view/__init__.py

```diff
@@ -0,0 +1,4 @@
+# Copyright 2019 MuK IT GmbH - Kerrim Abd El-Hamed
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from . import geo_raster_layer
```

Comment:
Please do not insert EOL at EOF

It ease further update of the file in commit history readability and printout of files won't concat last line of file 1 with first line of file 2 (for instance with `cat` in bash)